test/prod/compound/TestLifecycle.t.sol

```diff
@@ -143,10 +141,9 @@ contract TestLifecycle is TestSetup {
             supply.morphoUnderlyingBalanceBefore,
             string.concat(supply.market.symbol, " morpho balance")
         );
-        assertApproxEqAbs(
-            ICToken(supply.market.poolToken).balanceOfUnderlying(address(morpho)),
-            supply.morphoPoolSupplyBefore + supply.position.pool,
-            10,
+        assertTrue(
+            ICToken(supply.market.poolToken).balanceOfUnderlying(address(morpho)) >=
+                supply.morphoPoolSupplyBefore + supply.position.pool,
```

Comment:
```suggestion
        assertGe(
            ICToken(supply.market.poolToken).balanceOfUnderlying(address(morpho)),
                supply.morphoPoolSupplyBefore + supply.position.pool,
```

to also log values in case of error